pandas/core/arrays/_mixins.py

```diff
@@ -115,6 +117,13 @@ def T(self: _T) -> _T:
 
     # ------------------------------------------------------------------------
 
+    def equals(self, other) -> bool:
```

Comment:
what is hitting this one?

Reply:
This is now the implementation used by PandasArray.equals